examples/step-81/doc/intro.dox

```diff
@@ -160,11 +157,11 @@ $\mathbf{F}^\pm(\mathbf{x})=\lim_{\delta\to0,\delta>0}\mathbf{F}(\mathbf{x}\pm\d
 We will be using a rescaled version of the Maxwell's equations described above.
 The rescaling has the following key differences:<br />
 1. Every length is rescaled by the free-space wavelength $2\pi k^{-1}
-:= 2\pi(\omega\sqrt{\varepsilon_0\mu_0})^{-1}$, where $\varepsilon_0$ and $\mu_0$
+\dealcoloneq 2\pi(\omega\sqrt{\varepsilon_0\mu_0})^{-1}$, in which $\varepsilon_0$ and $\mu_0$
 denote the vacuum dielectric permittivity and magnetic permeability, respectively.
 <br />
 2. $\mathbf{E}$, $\mathbf{H}$, $\mathbf{J}_a$, $\mathbf{M}_a$ are all rescaled by
-typical electric current strength $J_0$, where $J_0$ is the strength of the
+typical electric current strength $J_0$, i.e., the strength of the
 prescribed dipole source at location $a$ in the $e_i$ direction in Cartesian
 coordinates.
 @f[
```

Comment:
One more thing: I have no idea what this equation (on line 168) is for. AFAICT it isn't used anywhere else and there's no explanatory text around it (nor is `\delta` ever defined).